### PR TITLE
Error fixed 

### PR DESCRIPTION
**Pull request by AlexeiTimbro**
Error fixed

### CODE REVIEW

**Review by Abdur-Siddiqui — APPROVED** [review]

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ function AdminCreateCoachNotes() {
             content_FR: newCoachNote.content_FR,
         };
         try {
-            const response = await fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/coachnotes`, {
+            const response = await fetch(`${process.env.REACT_APP_DEVELOPMENT_URL}/api/v1/coachnotes`, {
                 method: "POST",
                 headers: {
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js:81]: forgot to change the url

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ function AdminCreateCoachNotes() {
             content_FR: newCoachNote.content_FR,
         };
         try {
-            const response = await fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/coachnotes`, {
+            const response = await fetch(`${process.env.REACT_APP_DEVELOPMENT_URL}/api/v1/coachnotes`, {
```
Comment (Eli293) [inline fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js:79]: Why are you using development URL here ?

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js (hunk 1 above)
Comment (AlexeiTimbro) [reply fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js:81]: your right hold up

**Review by Abdur-Siddiqui — APPROVED** [review]

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js (hunk 2 above)
Comment (AlexeiTimbro) [reply fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js]: i just fixed it

**Review by emilegirardGit — APPROVED** [review]

**Pull request merged**